### DISCUSSION TITLE
Thoughts on post structure

### DISCUSSION THREAD

**Discussion by Bernadetadad** [dms-vep/dms-vep.github.io#13] [Ideas]
I've pushed [SC2_update_220124](https://github.com/dms-vep/dms-vep.github.io/tree/SC2_update_220124) branch to which I've added a new post with XBB.1.5 data. I did not spend much time comprehensively making the post as it just serves as a trial to test out how to make posts and how things look. Here are some thoughts after doing this:

- Installing node.js took some minor troubleshooting. @WillHannon-MCB add somewhere to contribution page that this should not be done on cluster
- after installation was done everything else worked great and being able to preview local version is super nice and was very easy to do
- I do think we need some better way to draw people's attention to key data. Now if you go to SARS-CoV-2 page it just looks too much like a blog without any hierarchical structure. I tried to indicate that this is the most up to date data in the `subtext` but it just doesn't stand out enough to make the point obvious. Perhaps this could be done by adding some frame or something around key posts to highlight them and pinning them to the top of the page?
- Ordering posts sequentially does not make practical sense, we want the most useful datasets to be by default at the top of the page and after those, depending on the project, we could have timeline ordered posts if people want to make these for some side-projects
-  I feel like the space on the blog posts is not used very well. Mostly for when altair plots are embedded they only take up a third of a page width and so for something like spike it basically means endless right scrolling to see parts of protein.

**Comment by jbloom** [comment]
I'm not sure how to look over this, but happy to meet in person to discuss once points @Bernadetadad mentions are addressed and I know how to look.

**Comment by WillHannon-MCB** [comment]
@Bernadetadad, I merged your branch into `main`.  @jbloom, you can see the example post [here](https://dms-vep.org/experiments/sars2/posts/XBB.1.5_ACE2_binding_serum_escape_cell_entry.html). 

Regarding each point: 

- I added a better description to the contributing guide to avoid issues with `Node`. I'm going to make a Docker container to make it easier to develop.
- I'm glad that the local preview worked for you!
- I'm working on a better design to draw people's attention to the key data sources. 
- Agreed, I'll think of a different way to order posts. I'll probably add a way to 'pin' datasets when you edit the 'blog' page. 
- I'll edit the CSS to give more space to the content on a larger screen so the Altair plots aren't getting cut off so much.

**Comment by WillHannon-MCB** [comment]
@Bernadetadad, I tried to make it so that there is more space for `Altair` plots on wide screens. I might need to tweak it to get the right balance of screen size. Let me know if this is better.

I changed the structure of the landing page for each virus. It's not a huge change and might not be the final design, but it should address some of the problems you mentioned. Check it out [here](https://dms-vep.org/experiments/sars2/).

Right now it's only an outline of what it could look like. If you want to add to it, I added instructions for that [here](https://dms-vep.org/contributing.html#updating-the-landing-page). I'm also happy to fill it out if you have particular suggestions!
